lib/bibliothecary/analyser.rb

```diff
@@ -24,6 +24,7 @@ def self.create_analysis(platform_name, relative_path, kind, dependencies)
     def self.included(base)
       base.extend(ClassMethods)
     end
+
     module ClassMethods
       def generic?
         platform_name == "generic"
```

Comment:
I'm not sure when this "generic?" predicate is used if at all, but I wonder whether when it's used, someone would be expecting it to also match cyclonedx. It's possible I added it thinking it would enable special-casing a multi-platform dependencies.csv and then ended up not succeeding in using this.

Reply:
so one option could be (maybe in its own PR) make dependencies.csv work the same as cyclonedx does now and drop this "generic " idea.